src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -591,4 +615,33 @@ protected function note($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * Set the event dispatcher instance.
+     *
+     * @param  \Illuminate\Contracts\Events\Dispatcher  $dispatcher
+     * @return void
+     */
+    public function setEventDispatcher(Dispatcher $dispatcher)
+    {
+        static::$dispatcher = $dispatcher;
```

Comment:
```suggestion
        $this->dispatcher = $dispatcher;
```

Reply:
Should this be completely removed now, since it should be handled by the IoC resolver?

Reply:
Done